docker-compose-dev.yaml

```diff
@@ -1,99 +1,99 @@
     external: false
 
-  anchore-scratch: {}
-
 services:
   # The primary API endpoint service
-  engine-api:
-    image: anchore/anchore-engine:dev
+  api:
+    image: anchore/anchore-engine-dev:latest
     depends_on:
-    - anchore-db
-    - engine-catalog
+    - db
+    - catalog
     ports:
     - "8228:8228"
     logging:
       driver: "json-file"
       options:
         max-size: 100m
     environment:
-    - ANCHORE_ENDPOINT_HOSTNAME=engine-api
-    - ANCHORE_DB_HOST=anchore-db
+    - ANCHORE_ENDPOINT_HOSTNAME=api
+    - ANCHORE_DB_HOST=db
     - ANCHORE_DB_PASSWORD=mysecretpassword
     command: ["anchore-manager", "service", "start", "apiext"]
 
   # Catalog is the primary persistence and state manager of the system
-  engine-catalog:
-    image: anchore/anchore-engine:dev
+  catalog:
+    image: anchore/anchore-engine-dev:latest
     depends_on:
-    - anchore-db
+    - db
     logging:
       driver: "json-file"
       options:
         max-size: 100m
     expose:
     - 8228
     environment:
-    - ANCHORE_ENDPOINT_HOSTNAME=engine-catalog
-    - ANCHORE_DB_HOST=anchore-db
+    - ANCHORE_ENDPOINT_HOSTNAME=catalog
+    - ANCHORE_DB_HOST=db
     - ANCHORE_DB_PASSWORD=mysecretpassword
     command: ["anchore-manager", "service", "start", "catalog"]
-  engine-simpleq:
-    image: anchore/anchore-engine:dev
+  queue:
+    image: anchore/anchore-engine-dev:latest
     depends_on:
-    - anchore-db
-    - engine-catalog
+    - db
+    - catalog
     expose:
     - 8228
     logging:
       driver: "json-file"
       options:
         max-size: 100m
     environment:
-    - ANCHORE_ENDPOINT_HOSTNAME=engine-simpleq
-    - ANCHORE_DB_HOST=anchore-db
+    - ANCHORE_ENDPOINT_HOSTNAME=queue
+    - ANCHORE_DB_HOST=db
     - ANCHORE_DB_PASSWORD=mysecretpassword
     command: ["anchore-manager", "service", "start", "simplequeue"]
-  engine-policy-engine:
-    image: anchore/anchore-engine:dev
+  policy-engine:
+    image: anchore/anchore-engine-dev:latest
     depends_on:
-    - anchore-db
-    - engine-catalog
+    - db
+    - catalog
     expose:
     - 8228
     logging:
       driver: "json-file"
       options:
         max-size: 100m
     environment:
-    - ANCHORE_ENDPOINT_HOSTNAME=engine-policy-engine
-    - ANCHORE_DB_HOST=anchore-db
+    - ANCHORE_ENDPOINT_HOSTNAME=policy-engine
+    - ANCHORE_DB_HOST=db
     - ANCHORE_DB_PASSWORD=mysecretpassword
     command: ["anchore-manager", "service", "start", "policy_engine"]
-  engine-analyzer:
-    image: anchore/anchore-engine:dev
+  analyzer:
+    image: anchore/anchore-engine-dev:latest
     depends_on:
-    - anchore-db
-    - engine-catalog
+    - db
+    - catalog
     expose:
     - 8228
     logging:
       driver: "json-file"
       options:
         max-size: 100m
     environment:
-    - ANCHORE_ENDPOINT_HOSTNAME=engine-analyzer
+    - ANCHORE_ENDPOINT_HOSTNAME=analyzer
     - ANCHORE_DB_HOST=anchore-db
```

Comment:
missed a anchore-db that needs to change to db

Reply:
:+1: